README.md

```diff
@@ -629,6 +629,7 @@ There are three kinds of matchers:
 | `timestamp` | A datetime in any recognized format | |
 | `token` | Any sequence of non-whitespace characters | `1+x$3` |
 | `trailingident` | Multiline content with indented trailing lines | |
+| `unixdt` | A datetime in Unix time format (seconds since Unix epoch) | `1608694199.999` |
```

Comment:
Should this note the seconds/milliseconds cut-off?

Reply:
Agreed. I will add clarification.